code/modules/paperwork/photocopier.dm

```diff
@@ -203,6 +221,41 @@
 		visible_message("<span class='notice'>A red light on \the [src] flashes, indicating that it is out of toner.</span>")
 	return p
 
+
+/obj/machinery/photocopier/proc/copyass()
+	var/icon/temp_img
+	if(ishuman(ass) && (ass.get_item_by_slot(slot_w_uniform) || ass.get_item_by_slot(slot_wear_suit)))
+		usr << "<span class='notice'>You feel kind of silly copying [ass == usr ? "your" : ass][ass == usr ? "" : "\'s"] ass with [ass == usr ? "your" : "their"] clothes on.</span>"
+		return
+	if(check_ass()) //You have to be sitting on the copier and either be a xeno or a human without clothes on.
+		if(ishuman(ass)) //Suit checks are in check_ass
+			var/mob/living/carbon/human/H = ass
+			temp_img = icon('icons/obj/butts.dmi', "[H.species.butt]")
```

Comment:
This doesn't seem like a good way to go about it, personally; it would probably be better to H.get_species() and just ensure that the pictures are properly named what get_species() would return.

There shouldn't really be an entire butt var just because of copy machines.